src/gui/guiFormSpecMenu.cpp

```diff
@@ -1220,12 +1322,34 @@ void GUIFormSpecMenu::parseLabel(parserData* data, const std::string &element)
 			// than multiply by 0.4, to get exact results
 			// in the integer cases: 0.4 is not exactly
 			// representable in binary floating point.
-			s32 posy = pos.Y + ((float)i) * spacing.Y * 2.0 / 5.0;
 			std::wstring wlabel = utf8_to_wide(unescape_string(lines[i]));
-			core::rect<s32> rect = core::rect<s32>(
-				pos.X, posy - m_btn_height,
-				pos.X + m_font->getDimension(wlabel.c_str()).Width,
-				posy + m_btn_height);
+
+			core::rect<s32> rect;
+
+			if (data->real_coordinates) {
+				v2s32 pos = getRealCoordinateBasePos(false, &v_pos);
+
+				// Add offset for smaller rect.
+				pos.Y += (imgsize.Y / 4) + ( ((float)imgsize.Y) * i / 2);
```

Comment:
```suggestion
				pos.Y += (imgsize.Y / 4) + (((float)imgsize.Y) * i / 2);
```

Is it good to change the line distance? Before, it was `spacing.Y * 2.0 / 5.0 = imgsize.Y * 15/13 *  2/5 = imgsize.Y * 6 / 13` which is a bit less than `imgsize.Y / 2`.
Probably it is good. But you should change the code comments up there.

Reply:
`imgsize.Y / 2` will line up with the new elements properly so that newlines can be used instead of multiple label elements with ease.  Just `imgsize` would have lines that are too far apart, and if the person decided to use elements spaced half the normal height (which is probably a likely occurrence, since spacings of one is too big and anything else is tricky to work with), then a whole `imgsize` wouldn't line up at all.

The old spacing, though, doesn't make sense at all with the new system, which is why I am changing it.